probe-rs-tools/Cargo.toml

```diff
@@ -14,6 +14,17 @@ description = "A collection of on chip debugging tools to communicate with micro
 
 default-run = "probe-rs"
 
+[features]
+default = []
+# Include server/client functionality
+remote = [
```

Comment:
in my opinion `remote` should be in the default flags or at least be included in the release binary.

Reply:
I'd probably prefer keeping it optional until we manage to hammer out its kinks